doc/ontodoc/update-pages.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# Bash script for generating documentation
+set -e
+set -x
+
+# Directories
+rootdir=$(git rev-parse --show-toplevel)
+ontodocdir=$rootdir/doc/ontodoc
+tmpdir=$ontodocdir/tmp
+pagesdir=$tmpdir/gh-pages
+
+# Generate documentation
+$ontodocdir/mkdoc.sh
+
+# Check up gh-pages
+if ! [ -d $pagesdir ]; then
+    git clone --branch=gh-pages --single-branch \
+        git@github.com:BIG-MAP/OntoBATT.git $pagesdir
+fi
+
+# Copy documentation to gh-pages
+# FIXME - generate separate index.html with links to versions
+cd $pagesdir
+cp -u $tmpdir/battinfo.html index.html
+
+# Update gh-pages
+git pull origin gh-pages
+git add index.html
```

Comment:
I don't have the complete overview of this doc generation and update workflow, but do you think there will be a case where the generated `index.html` has not changed and this line might fail? Or is it never present to begin with? If so, then ignore this comment.
It's just that `git add` will return a non-zero code if it didn't add anything.

Reply:
That is possible. I have encapsulated it in a if statement. Thanks